### PR TITLE
Prepare release of SDK 1.6.0

### PR DESCRIPTION
**Pull request by bertrand-lorentz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,48 +1,33 @@
...
-* Added fields to allow Group Framework Re-estimated Value BT-1561-NoticeResult to be unpublished.
-* Place of performance fields for lots are now aligned with parts and procedures.
-* BT-738-notice preferred publication date was added to forms X01 and X02.
-* Accessibility code and justification BT-754 and BT-755 accessibility now share the same repeatable parent node.
-* Renewal description BT-57-Lot, a multilingual field, now has its direct parent as parent node.
-* Framework Maximum value BT-271 was added to the notice type definitions.
-* Non-disclosure agreement fields BT-801 and BT-802 were added to the notice type definitions and the same parent node
-* Strategic procurement code and description BT-06 and BT-777 Strategic procurement now share the same parent node.
-* Removed BT-630 from notice subtype 8 in the notice type definitions.
+A comprehensive list of changes between SDK 1.5.1 and SDK 1.6.0 can be seen at <https://github.com/OP-TED/eForms-SDK/compare/1.5.1...1.5.0>
```
Comment (rouschr) [inline CHANGELOG.md:33]: I suppose the URL must be updated to 1.5.1...1.6.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,48 +1,33 @@
...
-# SDK 1.5.0 Release Notes
+Schematron rule now indicate the identifier of the field or node to which the rule applies, and this information is present in validation reports.
 
-This release of the SDK does not contain any backwards incompatible changes: software that was able to use version 1.4.0 should also be able to use this version.
+## Updated metadata content
 
 This version brings various changes in the metadata content that were needed for corrections, enhancements and consistency:
 
-* Several elements in the eForms schema were updated, to remove incorrect restrictions.
+* The content of codelists were updated.
```
Comment (rouschr) [inline CHANGELOG.md:21]: I think it would be a bit better to write: 
"The content of codelists was updated" (The content = it = it was updated)

**Review by rouschr — CHANGES_REQUESTED** [review]
I request changes because of that one link.

File: CHANGELOG.md (hunk 1 above)
Comment (bertrand-lorentz) [reply CHANGELOG.md]: Fixed, thanks !

File: CHANGELOG.md (hunk 2 above)
Comment (bertrand-lorentz) [reply CHANGELOG.md]: Corrected.

**Review by YvesJo — DISMISSED** [review]

**Review by pdonohoe — APPROVED** [review]

**Review by YvesJo — DISMISSED** [review]

**Review by pdonohoe — APPROVED** [review]

**Review by YvesJo — DISMISSED** [review]

**Review by ferrakaop — APPROVED** [review]

**Review by rouschr — APPROVED** [review]

**Pull request merged**